### PR TITLE
Update cookbook for PAPO and some typos

### PR DESCRIPTION
**Pull request by LaureBro**
So far we still say that they need to contact the support to get an EE archive.

### CODE REVIEW

File: developer_guide/migration/index.rst [hunk 1]
```diff
@@ -96,11 +96,12 @@ Here are the migration guides:
 
 **Enterprise Edition**
 
-Please contact our team through your access to the support service desk to ask for a standard archive.
+Standard EE archives are now available on a dedicated portal, pleae contact your sales manager or channel partner to have access to this portal if it's not already the case. You will be able to download your EE archive directly from the Portal interface.
```
Comment (grena) [inline developer_guide/migration/index.rst]: please*

**Comment by grena** [comment]
typo them GTM :)

**Comment by LeoBenoist** [comment]
GTM

File: developer_guide/migration/index.rst (hunk 1 above)
Comment (pandino101) [inline developer_guide/migration/index.rst]: "channel partner" or "channel manager"?
"to have access to" could be "to get access to"
"to this portal" could be "to it"
"if it's" should be "if it is". I would suggest not to use contractions on docs.

**Comment by pandino101** [comment]
I made some comments.

**Pull request merged**